### ISSUE TITLE
Allow setting "delete pull request branch branch after merge by default" as a global option for new repos

### ISSUE THREAD

**Issue by pat-s** [go-gitea/gitea#26669]
### Feature Description

AFAICS this is currently not supported through global settings judging from https://docs.gitea.com/next/administration/config-cheat-sheet#repository-repository

### Screenshots

![image](https://github.com/go-gitea/gitea/assets/8430564/5109b649-ff01-4b19-83c1-a0ec4957141f)

**Comment by TheFriendlyCoder** [comment]
I agree - this would be a very helpful option to be able to set globally.

**Comment by silverwind** [comment]
The global `[repository.pull-request] DEFAULT_DELETE_BRANCH_AFTER_MERGE` setting now supplies the saved default for newly created repositories.

Fixed in https://github.com/go-gitea/gitea/commit/ef88cdb7e73fdb626d3b8ce4c7efc938f51c8fea.

Written by Codex.
